src/store/BlueprintSlice.ts

```diff
@@ -0,0 +1,53 @@
+import { PayloadAction, createSlice } from '@reduxjs/toolkit';
+
+import { RootState } from '.';
+
+type blueprintsState = {
+  selectedBlueprint: string | undefined;
+  searchInput?: string;
+  offset?: number;
+  limit?: number;
+};
+
+const initialState: blueprintsState = {
+  selectedBlueprint: undefined,
+  searchInput: undefined,
```

Comment:
Let's initialize this as `''`.

Reply:
When the backend get empty string (`''`)  in the search  param,  it will do an empty searching...

Reply:
Yeah, lets not xD or fix the backend first :)

Reply:
Fair enough :smile: